src/CMakeLists.txt

```diff
@@ -471,21 +471,23 @@ if(WITH_CRIMSON)
       build_dpdk(${CMAKE_BINARY_DIR}/src/dpdk)
     endif()
   endif()
-  list(APPEND Seastar_CXX_FLAGS
-    "-DSEASTAR_NO_EXCEPTION_HACK"
-    "-Wno-error"
-    "-Wno-sign-compare"
-    "-Wno-attributes"
-    "-Wno-pessimizing-move"
-    "-Wno-address-of-packed-member"
-    "-Wno-non-virtual-dtor")
-  set(Seastar_CXX_FLAGS "${Seastar_CXX_FLAGS}" CACHE STRING "" FORCE)
   add_subdirectory(seastar)
   # create the directory so cmake won't complain when looking at the imported
   # target: Seastar exports this directory created at build-time
   file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/seastar/gen/include")
   file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/seastar/gen/src")
-  target_compile_options(seastar PUBLIC "-Wno-non-virtual-dtor")
+  target_compile_options(seastar
+  PUBLIC
+    "-Wno-non-virtual-dtor"
+  PRIVATE
+    "-DSEASTAR_NO_EXCEPTION_HACK"
+    "-Wno-error"
+    "-Wno-sign-compare"
+    "-Wno-attributes"
+    "-Wno-pessimizing-move"
+    "-Wno-address-of-packed-member")
+  get_target_property(Seastar_CXX_FLAGS seastar COMPILE_OPTIONS)
+  message(STATUS "Seastar_CXX_FLAGS: ${Seastar_CXX_FLAGS}")
```

Comment:
i'd prefer dropping this line. we do know what we are doing.

Reply:
> It solely for informative purposes, I'm ok with dropping/commenting it out as well.
> 

please.

> > but if that's the case, why adding them twice?
> 
> Not sure I understand, I've used _get_ target property just so we could print them.

`Seastar_CXX_FLAGS` is a setting for building Seastar, see https://github.com/scylladb/seastar/blob/faa171e30cfa3fb4efa212a28f52792a150d6bf6/CMakeLists.txt#L225 . it'd be confusing if we both

- set these compile flags using ` target_compile_options(seastar ...)` *and*
- set them with `Seastar_CXX_FLAGS` 

despite that the latter does not take effect, as it is set after this variable is read by `src/seastar/CMakeLists.txt`. but it's still confusing. and IMHO, also not necessary.